clang/test/CodeGen/fp-accuracy.c

```diff
@@ -339,6 +422,28 @@ void f1(float a, float b) {
 // CHECK-F4: call void @llvm.fpbuiltin.sincos.f64(double {{.*}}, ptr {{.*}}, ptr {{.*}}) #[[ATTR_F4_MEDIUM]]
 // CHECK-F4: call float @tanf(float {{.*}})
 //
+// CHECK-F5-LABEL: define dso_local void @f2(
+// CHECK-F5: call float @llvm.cos.f32(float {{.*}})
+// CHECK-F5: call float @llvm.sin.f32(float {{.*}})
+// CHECK-F5: call double @llvm.fpbuiltin.tan.f64(double {{.*}}) #[[ATTR_F5_HIGH]]
+// CHECK-F5: call double @llvm.log10.f64(double {{.*}})
+// CHECK-F5: call i32 (double, ptr, ptr, ...) @sincos(double noundef {{.*}}, ptr noundef {{.*}}, ptr noundef {{.*}})
+// CHECK-F5: call float @tanf(float noundef {{.*}})
+//
+// CHECK-F5: attributes #[[ATTR_F5_MEDIUM]] = {{.*}}"fpbuiltin-max-error"="4.0"
+// CHECK-F5: attributes #[[ATTR_F5_HIGH]] = {{.*}}"fpbuiltin-max-error"="1.0"
+//
+// CHECK-F6-LABEL: define dso_local void @f2(
+// CHECK-F6: call float @llvm.fpbuiltin.cos.f32(float {{.*}}) #[[ATTR_F6_MEDIUM]]
+// CHECK-F6: call float @llvm.fpbuiltin.sin.f32(float {{.*}}) #[[ATTR_F6_MEDIUM]]
```

Comment:
I may still be confused, so please bear with me.
F6 is: "-ffp-builtin-accuracy=medium high:[sin,atan]"
Shouldn't sin be High for line 438?

Reply:
The reason is because there is a call to "sinf" in f2, not "sin". The user is asking for "sin" to have accuracy "high".